frontend/src/components/output/label.jsx

```diff
@@ -28,16 +28,21 @@ class LabelOutput extends BaseComponent {
           </div>
         );
       }
+      return (
+        <div className="output_label">
+          <div className="output_class">{this.props.value["label"]}</div>
+          <div className="confidence_intervals">
+            <div className="labels" style={{ maxWidth: "120px" }}>
+              {labels}
+            </div>
+            <div className="confidences">{confidences}</div>
+          </div>
+        </div>
+      );        
     }
     return (
-      <div className="output_label">
```

Comment:
all the component class names follow a convention, basically "[input|output]_$componentname". Can you keep the top level class output_label, and add a class to output class?